src/test/java/org/tikv/txn/TwoPhaseCommitterTest.java

```diff
@@ -76,4 +99,170 @@ public void autoClosableTest() throws Exception {
             executorService)) {}
     Assert.assertTrue(executorService.isShutdown());
   }
+
+  @Test
+  public void prewriteWriteConflictFastFailTest() throws Exception {
```

Comment:
It seems this test has nothing to do with this pr, right?

Reply:
This test is based on the new constructor method added in this PR https://github.com/tikv/client-java/pull/775. 
Do you mind move it to that PR, or the test will fail

Reply:
this test for verify write conflict will aways retry ,but need fast fail

Reply:
could you explain why this test was added? Or what scenes/feature this test for